### PR TITLE
Add kafka-exporter to expose prometheus metrics for kafka cluster

### PR DESCRIPTION
**Pull request by NavinSharma13**
**Add kafka-exporter to expose prometheus metrics for kafka cluster**

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: charts/datahub/values.yaml [hunk 1]
```diff
@@ -111,6 +111,11 @@ datahubUpgrade:
         cpu: 300m
         memory: 256Mi
 
+prometheus-kafka-exporter:
+  enabled: false
+  kafkaServer:
+  - prerequisites-kafka:9092  # <<release-name>>-kafka:9092
```
Comment (pedro93) [inline charts/datahub/values.yaml:117]: We already have this value specified under `global.kafka.bootstrap.server`, can we not re-use that variable?

Comment (szalai1) [reply charts/datahub/values.yaml:117]: I think it should be configured here: https://github.com/acryldata/datahub-helm/blob/master/charts/datahub/values.yaml

Comment (NavinSharma13) [reply charts/datahub/values.yaml:117]: we are using kafka-exporter directly from repository: https://prometheus-community.github.io/helm-charts  similar to schema-registry .So we need to define values under chart name

File: charts/datahub/values.yaml [hunk 2]
```diff
@@ -111,6 +111,71 @@ datahubUpgrade:
...
+  # - name: <ENV_VAR_NAME>
+  #   value: <value>
+
+  # List of additional cli arguments to configure kafka-exporter
+  # for example: --log.enable-sarama, --log.level=debug, etc.
+  # all the possible args can be found here: https://github.com/danielqsj/kafka_exporter#flags
+  extraArgs: []
+  
+  prometheus:
+  serviceMonitor:
```
Comment (szalai1) [inline charts/datahub/values.yaml:131]: is indentation right here?

Comment (pedro93) [reply charts/datahub/values.yaml:131]: Good catch, @NavinSharma13 please match indentation like this: https://github.com/prometheus-community/helm-charts/blob/88447d1886a982b0e26f047d9973d2f6bad7fe7e/charts/prometheus-kafka-exporter/values.yaml#L64

File: charts/datahub/values.yaml [hunk 3]
```diff
@@ -111,6 +111,71 @@ datahubUpgrade:
...
+  sarama:
+    logEnabled: false
+  env: []
+  # - name: <ENV_VAR_NAME>
+  #   value: <value>
+
+  # List of additional cli arguments to configure kafka-exporter
+  # for example: --log.enable-sarama, --log.level=debug, etc.
+  # all the possible args can be found here: https://github.com/danielqsj/kafka_exporter#flags
+  extraArgs: []
```
Comment (pedro93) [inline charts/datahub/values.yaml:128]: Why do we need all these parameters written out explicitly? Please only add what you need to change for Kafka-exporter in data hub's deployment. If most things are default, then please omit them. This is adding quite a bit to an already large values.yaml file

**Review by pedro93 — CHANGES_REQUESTED** [review]

File: charts/datahub/values.yaml (hunk 2 above)
Comment (NavinSharma13) [reply charts/datahub/values.yaml]: updated in new commit

**Pull request merged**